README.md

```diff
@@ -137,7 +132,6 @@ We do NOT embed any i18n data within our library. However, we make it really
 easy to use. Read [How to get and load CLDR JSON data](#cldr) for more
```

Comment:
Another pull request is not needed. You can simply push one more commit to this same branch.